### PR TITLE
feat: node v0.7.0-rc9 adaptations

### PR DESCRIPTION
**Pull request by Bidon15**
## Overview

To make testground tests compatible with latest arabica/robusta testnets

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: testkit/nodekit/node.go [hunk 1]
```diff
@@ -35,14 +41,30 @@ func NewNode(path string, tp node.Type, network string, cfg *nodebuilder.Config,
...
+	ring, err := keyring.New(app.Name, cfg.State.KeyringBackend, keysPath, os.Stdin, encConf.Codec)
+	if err != nil {
+		return nil, err
+	}
+
+	store, err := nodebuilder.OpenStore(path, ring)
 	if err != nil {
 		return nil, err
 	}
 	return nodebuilder.NewWithConfig(tp, p2p.Network(network), store, cfg, options...)
```
Comment (jrmanes) [inline testkit/nodekit/node.go:57]: could you please remove this blank line? 🙏 
(sorry for my OCD haha 🥲)

**Review by jrmanes — APPROVED** [review]
LGTM 🚀

File: testkit/nodekit/node.go (hunk 1 above)
Comment (Bidon15) [reply testkit/nodekit/node.go]: Fixed 👍 
Thanks for the catch

**Comment by derrandz** [comment]
Hi! Awesome changes! I think `nodekit.IsSyncing` should return `!state Finished()` instead.

**Pull request merged**